glaring-client/src/hooks/useAppData.js

```diff
@@ -13,6 +13,21 @@ export default function useAppData() {
   });
 
 
+ async function cancel(id) {
```

Comment:
if the function works, that's good, but it needs a more specific name than 'cancel.' something like 'removeGroup' would be good.

Reply:
i would also really really suggest that we include a confirmation modal for deletes, as deletes are destructive and permanent and we don't want users to lose data on a misclick.